aspnetcore/fundamentals/routing.md

```diff
@@ -586,6 +586,72 @@ Complex segments (for example `[Route("/x{token}y")]`) are processed by matching
 <!-- While that code is no longer used by ASP.NET Core for complex segment matching, it provides a good match to the current algorithm. The [current code](https://github.com/aspnet/AspNetCore/blob/91514c9af7e0f4c44029b51f05a01c6fe4c96e4c/src/Http/Routing/src/Matching/DfaMatcherBuilder.cs#L227-L244) is too abstracted from matching to be useful for understanding complex segment matching.
 -->
 
+## Configuring endpoint metadata
+
+The following links provide information on configuring endpoint metadata:
+
+* [Enable Cors with endpoint routing](xref:security/cors#enable-cors-with-endpoint-routing)
+* [IAuthorizationPolicyProvider sample](https://github.com/aspnet/AspNetCore/tree/release/3.0/src/Security/samples/CustomPolicyProvider) using a custom `[MinimumAgeAuthorize]` attribute
+* [Test authentication with the [Authorize] attribute](xref:security/authentication/identity#test-identity)
+* <xref:Microsoft.AspNetCore.Builder.AuthorizationEndpointConventionBuilderExtensions.RequireAuthorization*>
+* [Selecting the scheme with the [Authorize] attribute](xref:security/authorization/limitingidentitybyscheme#selecting-the-scheme-with-the-authorize-attribute)
+* [Applying policies using the [Authorize] attribute](xref:security/authorization/policies#applying-policies-to-mvc-controllers)
+* <xref:security/authorization/roles>
+
+<a name="hostmatch"></a>
+
+## Host matching in routes
+
+The following code uses `RequireHost` to require the specified host on the route:
```

Comment:
The text given to require host/HostAttribute should be explained.

Can be:
* Host: `www.domain.com`
* Host with wildcard sub-domain: `*.domain.com`
* Host and port: `www.domain.com:5000` (or `*.domain.com:5000`)
* Port: `*:5000`

Reply:
@JamesNK  I changed wildcard to the following:

Host with wildcard sub-domain: `*.domain.com` (matches`domain.com`, `www.domain.com`, `subdomain.domain.com`, or `www.subdomain.domain.com` on any port)

Reply:
I'm not sure about multiple domains from a wildcard: `www.subdomain.domain.com`. Let me check the implementation.